aspnetcore/blazor/fundamentals/routing.md

```diff
@@ -553,7 +553,76 @@ Pass `true` to the `forceLoad` parameter to ensure a full-page reload is always
 Navigation.Refresh(true);
 ```
 
-To disable enhanced navigation and form handling, see <xref:blazor/fundamentals/startup#enhanced-navigation-and-form-handling>.
+Enhanced navigation is enabled by default, but it can be controlled hierarchically and on a per-link basis using the `data-enhance-nav` HTML attribute.
+
+The following examples disable enhanced navigation:
+
+```html
+<a href="redirect" data-enhance-nav="false">
+    GET without enhanced navigation
+</a>
+```
+
+```razor
+<ul data-enhance-nav="false">
+    <li>
+        <a href="redirect">GET without enhanced navigation</a>
+    </li>
+    <li>
+        <a href="redirect-2">GET without enhanced navigation</a>
+    </li>
+</ul>
+```
+
+If the destination is a non-Blazor endpoint, enhanced navigation doesn't apply, and the client-side JavaScript retries as a full page load. This ensures no confusion to the framework about external pages that shouldn't be patched into an existing page.
+
+Form POST requests adopt [enhanced navigation](xref:blazor/fundamentals/routing#enhanced-navigation-and-form-handling) for form POST requests with the *explicit addition* of the `Enhance` parameter to `EditForm` forms or the *explicit addition* of the `data-enhance` attribute to the HTML form (`<form>`):
```

Comment:
Is the "explicit addition" text trying to say that the `data-enhance` attribute can't be added dynamically, like with JS? If yes, should we just explicitly state that as an additional note?

```suggestion
To enable enhanced form handling, add the `Enhance` parameter to `EditForm` forms or the `data-enhance` attribute to HTML forms (`<form>`):
```